src/test/passes/arithmetic/report_diff-x86_64-linux.c

```diff
@@ -0,0 +1,17 @@
+// REQUIRES: x86-registered-target
+
+// RUN: env OMVLL_CONFIG=%S/report_diff.py clang -target x86_64-pc-linux-gnu -fpass-plugin=%libOMVLL -O1 -S %s -o /dev/null | FileCheck %s
+
+// CHECK: omvll::Arithmetic applied obfuscation
+// CHECK: --- original
+// CHECK: +++ obfuscated
+// CHECK: @@
+// CHECK: -
+// CHECK: +
```

Comment:
> It makes sure that the Python function report_diff got executed and printed some diff.

Where is the diff printed? Maybe worth to show just a few changes.

Reply:
This isn't supposed to check the diff itself, but rather check that the `report_diff` function in from the config script was called (which prints `omvll::Arithmetic applied obfuscation`) and we get some diff (`@@`, `+`, `-`, etc.).